src/microbe_stage/MicrobeWorldSimulation.cs

```diff
@@ -203,6 +206,8 @@ public void Init(Node visualDisplayRoot, CompoundCloudSystem cloudSystem)
         // Systems stored in properties
         CameraFollowSystem = new CameraFollowSystem(EntitySystem);
 
+        MicrobeProcessManagerSystem = new MicrobeProcessManagerSystem(EntitySystem, couldParallelize);
```

Comment:
I'd say this should parallelize itself as it looks to be doing as much computation as the process system.